Source/WebCore/inspector/agents/InspectorCSSAgent.cpp

```diff
@@ -916,6 +918,54 @@ Inspector::Protocol::ErrorStringOr<Ref<JSON::ArrayOf<String>>> InspectorCSSAgent
     return fontFamilyNames;
 }
 
+Inspector::Protocol::ErrorStringOr<Ref<JSON::ArrayOf<Inspector::Protocol::CSS::FontFace>>> InspectorCSSAgent::getCustomFonts()
+{
+    auto customFonts = JSON::ArrayOf<Inspector::Protocol::CSS::FontFace>::create();
+    auto* domAgent = m_instrumentingAgents.persistentDOMAgent();
+
+    for (auto* document : domAgent->documents()) {
```

Comment:
there's no guarantee that the `domAgent` will exist here, so id either wrap this entire `for` in a `if (auto* domAgent = m_instrumentingAgents.persistentDOMAgent()) { ... }` or add a `return` here in a `if (!domAgent)`

Reply:
I replaced with this:
```
if (auto* domAgent = m_instrumentingAgents.persistentDOMAgent()) {
        for (auto* document : domAgent->documents()) {
```